justfile

```diff
@@ -68,3 +68,4 @@ _lint-proto:
   buf format -d --exit-code
   buf breaking proto/executionapis --against 'buf.build/astria/execution-apis'
   buf breaking proto/sequencerapis --against 'buf.build/astria/astria'
+  buf breaking proto/composerapis --against 'buf.build/astria/astria'
```

Comment:
The new one will have to be against `astria/composer-apis` and you will need to update `.github/workflows/release.yml` to push these to buf.